### ISSUE TITLE
When blank link text include to " >" that published page give raise to bug.

### ISSUE THREAD

**Issue by kurudrive** [WordPress/gutenberg#14300]
**Describe the bug**
When blank link text include to " >"  that published page give raise to bug.

**To Reproduce**
1. fill in the follow code in code editor

```
<!-- wp:list -->
<ul><li>[ test ] <a href="https://google.com" target="_blank" rel="noreferrer noopener" aria-label="Write it like div > h1 {color:red} (opens in a new tab)">Write it like div &gt; h1 {color:red}</a></li></ul>
<!-- /wp:list -->

<!-- wp:paragraph -->
<p>This text id no display at published page.</p>
<!-- /wp:paragraph -->
```
2. Editor view is OK, but ...
3. Published page is break that ">" was interpreted as an end tag

**Screenshots**

![25%](https://vektor-inc.co.jp/github_src/blank_link_edit.png)

![25%](https://vektor-inc.co.jp/github_src/blank_link_publish.png)

**Desktop (please complete the following information):**
 - OS: 10.13.2
 - Browser Chrome
 - Version 72.0.3626.119

**Additional context**
- Please add the version of Gutenberg you are using in the description.

WordPress 5.1( ja )

**Comment by youknowriad** [comment]
Wasn't this fixed recently cc @ellatrix ?

**Comment by ellatrix** [comment]
Duplicate of #12683. There are other similar issues: #8779 and #9915. A fix was proposed in #9963 but was never merged.

**Comment by kurudrive** [comment]
@youknowriad @ellatrix 
Thank you for checking.
Sorry I let you use your time.
